composer.json

```diff
@@ -32,11 +32,15 @@
         "drupal/token_filter": "^2.1",
         "drupal/twig_tweak": "^3.3",
         "drupal/webform": "^6.2",
-        "drush/drush": "^12.4"
+        "drush/drush": "^12.4",
+        "guzzlehttp/guzzle": "^7.8"
     },
     "conflict": {
         "drupal/drupal": "*"
     },
+    "scripts": {
+        "phpcs": "phpcs --standard=Drupal --extensions=php,module,inc,install,test,profile,theme,css,info,txt,md"
+    },
```

Comment:
Are you planning on using this script? Based on initial state of that PR, you are not.
Maybe it will be better to set up GitHub actions for that. For sure you need to configure you phpstorm at very least